org.mume.MMapper.json

```diff
@@ -0,0 +1,148 @@
+{
+  "app-id": "org.mume.MMapper",
```

Comment:
how do i tell you control this? https://mume.org/

it seems like the website is for mume while the app is some client for it?

Reply:
https://mume.org/.well-known/org.flathub.VerifiedApps.txt

Reply:
@bbhtt please let me know if you need more proof. I wasn't able to generate a UUID yet under my Flathub account